lib/sidekiq/middleware/server/freekiqs.rb

```diff
@@ -29,15 +30,15 @@ def call(worker, msg, queue)
                 raise FreekiqException, ex.message
               end
             else
-              Sidekiq.logger.info { "Out of free kiqs for #{msg['class']} job #{msg['jid']}" }
+              Sidekiq.logger.info { "Out of freekiqs for #{msg['class']} job #{msg['jid']}" }
             end
           end
           raise ex
         end
 
         def get_freekiqs_if_enabled(worker, msg, ex)
           freekiqs = nil
-          if msg['retry']
```

Comment:
Tbh I don't 100% grok why you need to pull 'retry' from the class by 'retry_count' up above is still pulled from 'msg'. I trust that it works, however. Just pointing it out in case we can be more consistent or anything.